concordium-std/src/types.rs

```diff
@@ -1,8 +1,7 @@
-use concordium_contracts_common::{Amount, ExchangeRate};
+use concordium_contracts_common::{AccountBalance, Amount, ExchangeRate};
```

Comment:
We cannot break backwards compatibility, so this type should be re-exported here.